src/lib/components/organisms/ticker/style.module.scss

```diff
@@ -32,6 +38,27 @@
   }
 }
 
+.tickerScroll {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  scrollbar-width: none;
+  overflow-x: scroll;
+  column-gap: var(--space-2);
+  max-height: 100px;
+  transform: translateX(var(--ticker-offset));
+  transition: transform 0.5s ease-in-out;
+  width: auto;
+
+  &:-webkit-scrollbar {
+    display: none; /* for Chrome, Safari, and Opera */
+  }
+
+  @include mq(mobileLandscape) {
+    width: fit-content;
+  }
+}
+
 .tickerItem {
   width: var(--ticker-item-width);
```

Comment:
huh - yes - didn't change this in this PR. Is it possible for developers to override variables like this? Let me check that. 
I think it's worth keeping a default in the code so people can just use this component out of the box, but like the rest of the component lib, devs should definitely be able to amend. 

This is being used to help with the desktop page scrolling mechanism. So changing this would require changing how that works as well - possible but just something else to think about